example_package/README.md

```diff
@@ -0,0 +1,454 @@
+<!-- Jak nie umiesz otworzyć by ładniej wyglądało, tu jest czytelniej -->
+<!-- https://github.com/Stowarzyszenie-Talent/st-make/tree/main/example_package -->
+
+# Spis Treści
+
+- [Szablon paczki](#szablon-paczki)
+- [Ogólne informacje](#ogólne-informacje)
+- [doc](#doc)
+  - [talentTex.cls](#talenttexcls)
+- [prog](#prog)
+  - [Rozwiązania](#rozwiązania)
+  - [ingen](#ingen)
+  - [inwer](#inwer)
+  - [checkerka](#checkerka)
+  - [oi.h](#oih)
+    - [Scanner (Wczytywanie)](#scanner-wczytywanie)
+    - [CheckerVerdict](#checkerverdict)
+    - [checker_test](#checker_test)
+    - [InwerVerdict](#inwerverdict)
+    - [bug](#bug)
+    - [oi_assert](#oi_assert)
+    - [Random](#random)
+- [config.yml](#configyml)
+  - [Interactive tasks](#interactive-tasks)
+  - [Time](#time)
+  - [Memory](#memory)
+  - [Title](#title)
+  - [Scores](#scores)
+  - [Task ID](#task-id)
+  - [Contest type](#contest-type)
+  - [Expected scores](#expected-scores)
+- [in i out](#in-i-out)
+- [dlazaw](#dlazaw)
+
+# **Szablon paczki**
+
+Jest to przykładowy szablon paczki, który zalecamy używać.
+Jedynie dla zadań interaktywnych jest on inny.
+(Jeszcze nie zrobiliśmy dla niego szablonu).
+W przypadku chęci zrobienia takiego zadania obecnie najlepiej skontaktuj się z kimś kto wie jak to się robi.
+
+Aktualna wersja paczki zawsze znajduje się zawsze
+[na githubie](https://github.com/Stowarzyszenie-Talent/st-make/tree/main/example_package).
+`st-make init`, pobiera właśnie tą wersję.
+
+# **Ogólne informacje**
+
+1. Do kompilacji paczki używamy skryptu `st-make`.
+Jest on dostępny [na githubie](https://github.com/Stowarzyszenie-Talent/st-make).
+1. Każde zadanie ma `tag` - jest to najczęściej 3 literowy skrót.
```

Comment:
Najczęściej czy zawsze?

Reply:
najczęściej, sio przyjmuje też inne długości. Przyjęło się nieumownie że 3 to standard.